insights/client/utilities.py

```diff
@@ -235,7 +242,10 @@ def magic_plan_b(filename):
 
 def generate_container_id(container_name):
     # container id is a uuid in the namespace of the machine
-    return str(uuid.uuid5(uuid.UUID(generate_machine_id()), container_name.encode('utf8')))
```

Comment:
If this function is only called when `analyze_image` or `container_mode` are `True`, then why not always return `container_name.encode('utf8')`?  It seems like `generate_machine_id` would always return `None` in this case, so no need to do any of that.